src/components/Home.jsx

```diff
@@ -0,0 +1,35 @@
+import { Link } from "react-router-dom";
+import { useContext } from "react";
+import { UserContext } from "../contexts/UserContext";
+
+const Home = () => {
+  const { user, setUser } = useContext(UserContext);
+
+  return (
+    <nav>
+      <div>
+        <Link to="/articles">
+          <div>
+            <button>Articles</button>
+          </div>
+        </Link>
+      </div>
+      <p></p>
+      <div>
+        <Link to="/topics">
+          <div>
+            <button>Topics</button>
+          </div>
+        </Link>
+      </div>
+      <p></p>
```

Comment:
I don't love these random empty paragraphs - feels like you could achieve whatever you want to achieve with these through styling